vulnerability/db.go

```diff
@@ -0,0 +1,117 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package vulnerability
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/aquasecurity/trivy-db/pkg/metadata"
+	"github.com/elastic/elastic-agent-libs/logp"
+
+	"github.com/aquasecurity/trivy/pkg/commands/artifact"
+	"github.com/aquasecurity/trivy/pkg/flag"
+	trivy_types "github.com/aquasecurity/trivy/pkg/types"
+)
+
+type VulnerabilityDBManager struct {
```

Comment:
we usually try to avoid starting the name with the name of the package

Reply:
Also, do we want to export these structs if they are only used internally in the `vulnerability` package?